pkgs/development/python-modules/wheel/default.nix

```diff
@@ -1,41 +1,48 @@
-{ lib
-, setuptools
-, pip
-, buildPythonPackage
-, fetchFromGitHub
-, pytest
-, pytestcov
-, coverage
-, jsonschema
-, bootstrapped-pip
+{ lib, buildPythonPackage, fetchFromGitHub
+, bootstrapped-pip, setuptools
 }:
 
 buildPythonPackage rec {
   pname = "wheel";
-  version = "0.33.6";
+  version = "0.34.2";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "pypa";
     repo = pname;
     rev = version;
-    sha256 = "1bg4bxazsjxp621ymaykd8l75k7rvcvwawlipmjk7nsrl72l4p0s";
+    sha256 = "1mwh35ycv07ajnpcjc4rjdmndh6nyg03gdgag5m8c2af7z1xlcmj";
     name = "${pname}-${version}-source";
   };
 
-  checkInputs = [ pytest pytestcov coverage ];
-  nativeBuildInputs = [ bootstrapped-pip setuptools ];
+  nativeBuildInputs = [
+    bootstrapped-pip
+    setuptools
+  ];
 
-  catchConflicts = false;
```

Comment:
Will need to check whether this is still needed

Reply:
yeah, I got  a clean build on Darwin and Linux without ... but I don't have sufficient CPU to reasonably rebuild all dependencies so cannot guarantee.